requirements.txt

```diff
@@ -1,4 +1,46 @@
-Flask
-jsonify
-flask-cors
-requests
\ No newline at end of file
+appdirs==1.4.4
+astroid==2.3.3
+atomicwrites==1.4.0
+attrs==20.3.0
+beautifulsoup4==4.9.1
```

Comment:
Requirements.txt is a great idea to make it much easier to download all of the package dependencies locally!